src/commands/consent/pull-consent-metrics/impl.ts

```diff
@@ -57,7 +45,7 @@ export async function pullConsentMetrics(
           )}`,
       ),
     );
-    process.exit(1);
+    this.process.exit(1);
```

Comment:
best practice is to move environment stuff into the dependency-injected context (will also migrate e.g., `logger` into this model). this simplifies testing a lot, where we can more easily mock and spy on exit calls and logs. separately, will also move away from `process.exit()` in favor of just throwing, since process.exit() makes testing generally more complex